float32 channel_a
float32 channel_b
---
bool success